README.md

```diff
@@ -126,6 +126,18 @@ To build a Docker image, checkout a specific version of the repository and run:
 
 	docker build -t my:tag -f cmd/shfmt/Dockerfile .
 
+Alternatively, if you want an image that includes busybox, run:
+
+	docker build -t my:tag -f cmd/shfmt/Dockerfile --target busybox .
+
+To use the Docker image, run:
+
+	docker run --rm -v $PWD:/mnt -w /mnt my:tag <shfmt arguments>
+
+E.g.:
+
+	docker run --rm -v $PWD:/mnt -w /mnt shfmt:latest -i 2 -l .
```

Comment:
I think you can remove this second example. I don't want the README to be too long, and I think it's clear enough.

Reply:
done